### DISCUSSION TITLE
Jetbrains plugin: include "What's new" information

### DISCUSSION THREAD

**Discussion by emab** [biomejs/biome#647] [Ideas]
Small request, but it would be great to see some sort of summary of "What's new" when updating the plugin. Currently the tab is empty when updating.

**Comment by ematipico** [comment]
Would you like to help with this work? The plugin is fairly new, and so is the community around it.

**Reply by emab** [reply]
Yep definitely, still working on implementing a rule but happy to pick up some tickets for the plugin.
